add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
- Observations: The PHP code directly executes echo statements for response messages, which can lead to unstructured output handling and can complicate future transformations.
- Recommendations: Instead of directly echoing, consider using a Response class to handle HTTP responses or the built-in `http_response_code()` function.